apps/wallet-mobile/src/legacy/Dashboard/Dashboard.tsx

```diff
@@ -73,7 +80,12 @@ export const Dashboard = () => {
       )
       return
     }
-    setShowWithdrawalDialog(true)
+
+    openModal(
+      strings.warningModalTitle,
+      <WithdrawStakingRewards wallet={wallet} onSuccess={() => resetToTxHistory()} onCancel={() => closeModal()} />,
+      windowHeight - (20 / 100) * windowHeight,
```

Comment:
I think `windowHeight * 0.8` would be clearer

Reply:
yes you are right

`windowHeight - (20 / 100) * windowHeight = windowHeight * (1 -  0.2) = windowHeight * 0.8`

Reply:
```suggestion
      windowHeight * 0.8
```